functions/model/organization/tark/createOrg.js

```diff
@@ -59,6 +60,7 @@ exports.createOrg = functions.https.onRequest((request, response) => {
 
         const appDetailsUpdateJson = {
           TotalNumberOfOrganizations: totalNumberOfOrganizations + 1,
+          OrgDomains: admin.firestore.FieldValue.arrayUnion(orgDomain),
         };
```

Comment:
Its better to follow the approach we follow everywhere (i.e) push into the array and send it.

Reply:
resolved